pkg/core/cmdutil/flagutil/arguments.go

```diff
@@ -2,11 +2,10 @@
 package flagutil
 
 import (
-	"github.com/redhat-developer/app-services-cli/pkg/cmd/debug"
 	"github.com/spf13/pflag"
 )
 
 // AddDebugFlag adds the '--debug' flag to the given set of command line flags
 func AddDebugFlag(fs *pflag.FlagSet) {
```

Comment:
Wouldn't it be better to just remove this function and use VerboseFlag directly?